[2024-06-01 00:44:11,666] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 00:44:13,957] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 01:48:31,408] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 01:48:33,728] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 02:28:48,224] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 02:28:50,893] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 02:47:38,174] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 02:47:41,319] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 03:11:35,082] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 03:11:38,203] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 03:29:03,734] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 03:29:06,929] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 03:37:47,842] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 03:37:50,965] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 03:47:51,789] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 03:47:54,755] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 04:11:03,789] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 04:11:06,924] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 04:27:35,670] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 04:27:38,397] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 04:36:00,179] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 04:36:02,751] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 04:47:51,866] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 04:47:54,563] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 05:09:48,093] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 05:09:50,445] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 05:28:25,710] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 05:28:28,481] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 05:36:39,794] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 05:36:41,959] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 05:47:42,089] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 05:47:44,341] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 06:12:15,027] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 06:12:17,193] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 06:30:00,045] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 06:30:02,998] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 06:38:57,627] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 06:39:00,879] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 06:47:38,868] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 06:47:40,979] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 07:07:55,852] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 07:07:59,120] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 07:21:13,702] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 07:21:16,107] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 07:33:01,820] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 07:33:05,255] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 07:47:31,535] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 07:47:34,397] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 08:11:36,964] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 08:11:39,467] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 08:27:34,380] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 08:27:35,966] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 08:35:49,949] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 08:35:52,496] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 08:47:34,073] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 08:47:36,073] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 09:11:29,872] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 09:11:32,039] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 09:25:55,654] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 09:25:58,517] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 09:33:22,514] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 09:33:24,816] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 09:47:42,058] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 09:47:44,511] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 10:09:36,166] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 10:09:38,935] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 10:29:26,601] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 10:29:30,048] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 10:37:46,376] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 10:37:48,342] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 10:47:32,911] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 10:47:35,053] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 11:09:43,790] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 11:09:46,682] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 11:22:42,101] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 11:22:44,549] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 11:34:31,402] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 11:34:33,843] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 11:47:38,890] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 11:47:42,083] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 12:15:46,612] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 12:15:48,450] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 12:46:25,932] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 12:46:27,690] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 13:08:56,870] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 13:09:04,661] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 13:22:43,619] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 13:22:46,851] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 13:33:25,925] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 13:33:29,081] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 13:47:28,316] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 13:47:31,107] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 14:09:10,183] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 14:09:11,918] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 14:26:21,963] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 14:26:23,997] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 14:33:59,141] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 14:34:02,505] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 14:47:48,064] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 14:47:50,555] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 15:08:46,854] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 15:08:49,400] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 15:23:08,953] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 15:23:11,044] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 15:33:09,219] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 15:33:11,254] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 15:47:59,210] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 15:48:01,243] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 16:10:16,448] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 16:10:19,270] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 16:27:26,383] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 16:27:28,747] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 16:37:03,100] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 16:37:05,530] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 16:47:43,556] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 16:47:46,300] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 17:08:32,629] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 17:08:35,465] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 17:24:49,541] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 17:24:52,041] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 17:32:47,143] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 17:32:49,545] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 17:47:52,606] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 17:47:55,356] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 18:11:39,803] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 18:11:42,050] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 18:28:41,690] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 18:28:43,823] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 18:37:11,801] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 18:37:15,399] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 18:47:50,153] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 18:47:53,734] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 19:09:15,997] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 19:09:18,352] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 19:20:50,314] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 19:20:53,472] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 19:32:59,048] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 19:33:01,015] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 19:47:32,598] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 19:47:34,908] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 20:09:07,425] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 20:09:10,720] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 20:22:29,861] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 20:22:32,827] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 20:33:09,617] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 20:33:11,665] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 20:47:40,911] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 20:47:43,859] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 21:09:20,654] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 21:09:22,990] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 21:22:37,700] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 21:22:40,410] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 21:33:08,780] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 21:33:12,055] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 21:47:43,974] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 21:47:45,985] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 22:08:14,483] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 22:08:17,696] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 22:26:12,348] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 22:26:14,371] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 22:34:05,426] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 22:34:08,324] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 22:47:56,019] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 22:47:59,342] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 23:09:39,702] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 23:09:41,821] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 23:24:02,444] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 23:24:04,276] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 23:33:24,574] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 23:33:27,777] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-01 23:47:54,601] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-01 23:47:56,728] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 00:44:49,878] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 00:44:52,757] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 01:49:01,456] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 01:49:03,752] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 02:28:29,053] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 02:28:31,857] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 02:45:47,576] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 02:45:50,918] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 03:10:52,470] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 03:10:55,496] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 03:26:44,925] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 03:26:47,713] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 03:34:36,766] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 03:34:39,230] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 03:47:38,167] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 03:47:41,328] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 04:10:59,416] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 04:11:02,616] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 04:28:39,041] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 04:28:41,724] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 04:37:18,581] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 04:37:21,286] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 04:47:53,170] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 04:47:55,655] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 05:09:08,087] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 05:09:10,087] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 05:25:48,119] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 05:25:50,070] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 05:33:27,692] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 05:33:29,637] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 05:47:50,959] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 05:47:52,780] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 06:13:08,800] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 06:13:12,071] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 06:31:12,380] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 06:31:15,047] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 06:48:00,001] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 06:48:02,188] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 07:09:29,212] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 07:09:32,116] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 07:22:27,211] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 07:22:29,772] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 07:33:06,671] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 07:33:09,264] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 07:47:48,368] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 07:47:50,573] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 08:10:32,871] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 08:10:35,246] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 08:26:54,508] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 08:26:57,345] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 08:35:32,521] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 08:35:34,646] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 08:47:54,675] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 08:47:57,578] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 09:10:08,514] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 09:10:10,352] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 09:23:35,400] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 09:23:38,793] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 09:32:52,351] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 09:32:54,336] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 09:47:44,251] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 09:47:46,520] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 10:09:02,892] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 10:09:05,034] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 10:28:25,808] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 10:28:29,005] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 10:36:25,604] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 10:36:28,588] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 10:47:47,772] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 10:47:50,333] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 11:08:06,650] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 11:08:10,315] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 11:20:45,511] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 11:20:48,222] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 11:33:14,778] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 11:33:18,414] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 11:47:43,070] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 11:47:45,269] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 12:15:16,875] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 12:19:19,395] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 12:41:51,319] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 12:41:53,582] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 12:55:21,438] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 12:55:24,981] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 13:08:55,918] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 13:08:57,918] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 13:22:48,909] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 13:22:50,779] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 13:33:15,142] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 13:33:18,573] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 13:47:56,381] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 13:47:59,624] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 14:07:37,609] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 14:07:39,679] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 14:24:32,095] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 14:24:34,308] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 14:32:43,043] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 14:32:46,103] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 14:47:51,224] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 14:47:54,732] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 15:08:47,920] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 15:08:50,340] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 15:23:04,806] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 15:23:07,028] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 15:33:11,435] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 15:33:14,616] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 15:47:42,940] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 15:47:46,383] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 16:10:53,144] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 16:10:55,206] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 16:28:30,536] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 16:28:32,843] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 16:37:50,558] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 16:37:53,959] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 16:47:41,910] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 16:47:43,893] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 17:07:52,046] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 17:07:55,582] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 17:25:53,499] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 17:25:56,706] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 17:33:13,021] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 17:33:15,039] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 17:48:03,902] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 17:48:07,309] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 18:12:11,315] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 18:12:13,537] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 18:29:37,350] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 18:29:40,514] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 18:38:15,271] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 18:38:17,397] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 18:47:50,918] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 18:47:53,340] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 19:06:56,914] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 19:07:00,439] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 19:18:08,787] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 19:18:10,874] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 19:32:50,324] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 19:32:52,368] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 19:47:41,923] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 19:47:44,811] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 20:08:57,150] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 20:08:59,293] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 20:22:38,856] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 20:22:40,636] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 20:33:21,342] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 20:33:23,987] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 20:47:40,434] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 20:47:43,554] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 21:09:08,014] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 21:09:11,190] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 21:22:59,973] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 21:23:03,113] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 21:33:08,969] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 21:33:11,989] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 21:47:47,332] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 21:47:49,192] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 22:08:13,454] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 22:08:15,716] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 22:26:17,064] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 22:26:19,896] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 22:34:12,799] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 22:34:15,635] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 22:47:40,435] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 22:47:43,444] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 23:09:02,714] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 23:09:04,931] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 23:23:44,462] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 23:23:47,750] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 23:33:17,801] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 23:33:19,682] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-02 23:47:53,653] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-02 23:47:56,599] 	INFO 	102 	id_data.py 	<module>() 	TC ID Data Ended
[2024-06-03 00:42:33,973] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 00:42:34,282] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 01:46:56,221] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 01:46:56,513] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 02:27:22,721] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 02:27:23,014] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 02:44:11,104] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 02:44:11,513] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 02:51:30,899] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 02:51:31,166] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 03:11:22,290] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 03:11:22,628] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 03:28:02,919] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 03:28:03,117] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 03:36:39,935] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 03:36:40,202] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 03:47:51,433] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 03:47:51,724] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 04:13:45,430] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 04:13:45,737] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 04:30:26,038] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 04:30:26,313] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 04:39:48,728] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 04:39:49,041] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 04:47:32,624] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 04:47:32,930] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 05:10:59,942] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 05:11:00,314] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 05:28:23,838] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 05:28:24,199] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 05:36:34,374] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 05:36:34,682] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 05:47:35,834] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 05:47:36,128] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 06:13:16,495] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 06:13:16,762] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 06:31:51,309] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 06:31:51,652] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 06:47:43,845] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 06:47:44,160] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 07:09:22,280] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 07:09:22,484] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 07:21:51,788] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 07:21:52,052] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 07:33:25,491] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 07:33:25,842] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 07:47:57,930] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 07:47:58,254] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 08:13:26,460] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 08:13:26,733] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 08:30:04,603] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 08:30:04,899] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 08:39:33,761] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 08:39:34,059] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 08:47:44,608] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 08:47:44,984] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 09:10:26,329] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 09:10:26,638] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 09:25:17,316] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 09:25:17,608] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 09:33:22,714] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 09:33:23,024] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 09:47:36,021] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 09:47:36,325] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 10:12:05,487] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 10:12:05,767] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 10:32:12,626] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 10:32:12,984] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 10:47:39,586] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 10:47:39,925] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 11:08:14,816] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 11:08:15,093] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 11:21:10,424] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 11:21:10,745] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 11:33:26,761] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 11:33:27,177] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 11:47:50,572] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 11:47:50,783] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 12:17:41,522] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 12:17:41,824] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 12:50:30,709] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 12:50:31,022] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 13:11:36,877] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 13:11:37,138] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 13:26:57,255] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 13:26:57,459] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 13:34:23,259] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 13:34:23,550] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 13:47:37,702] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 13:47:37,996] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 14:09:20,492] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 14:09:20,814] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 14:27:36,294] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 14:27:36,609] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 14:36:09,912] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 14:36:10,229] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 14:47:47,091] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 14:47:47,414] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 15:10:27,416] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 15:10:27,627] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 15:25:33,978] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 15:25:34,285] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 15:33:18,633] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 15:33:18,859] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 15:47:43,863] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 15:47:44,155] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 16:12:26,193] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 16:12:26,621] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 16:31:53,339] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 16:31:53,768] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 16:47:46,339] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 16:47:46,655] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 17:09:02,083] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 17:09:02,444] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 17:25:04,068] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 17:25:04,386] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 17:32:53,173] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 17:32:53,459] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 17:47:44,893] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 17:47:45,192] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 18:12:43,005] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 18:12:43,226] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 18:31:57,638] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 18:31:58,064] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 18:47:53,294] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 18:47:53,612] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 19:09:00,927] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 19:09:01,136] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 19:20:05,487] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 19:20:05,830] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 19:34:39,690] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 19:34:40,004] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 19:47:42,693] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 19:47:42,996] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 20:09:47,473] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 20:09:47,793] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 20:24:26,933] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 20:24:27,237] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 20:34:25,759] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 20:34:26,077] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 20:47:44,430] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 20:47:44,806] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 21:10:00,401] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 21:10:00,708] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 21:23:07,868] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 21:23:08,187] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 21:34:11,264] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 21:34:11,562] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 21:47:42,638] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 21:47:42,843] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 22:09:43,678] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 22:09:43,980] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 22:27:24,603] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 22:27:24,901] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 22:35:56,966] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 22:35:57,231] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 22:48:01,520] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 22:48:01,961] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 23:09:22,011] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 23:09:22,343] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 23:23:55,761] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 23:23:56,154] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 23:33:30,976] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 23:33:31,158] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-03 23:47:41,857] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-03 23:47:42,139] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 00:41:55,816] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 00:41:56,186] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 01:45:30,553] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 01:45:30,883] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 02:22:11,893] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 02:22:12,188] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 02:43:01,900] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 02:43:02,217] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 02:50:16,759] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 02:50:17,084] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 03:11:41,871] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 03:11:42,274] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 03:28:02,958] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 03:28:03,362] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 03:36:19,950] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 03:36:20,229] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 03:47:42,973] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 03:47:43,283] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 04:11:16,667] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 04:11:16,887] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 04:27:52,052] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 04:27:52,413] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 04:36:32,541] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 04:36:32,751] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 04:48:09,933] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 04:48:10,240] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 05:12:02,540] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 05:12:02,811] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 05:30:56,184] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 05:30:56,378] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 05:47:33,385] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 05:47:33,698] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 06:13:11,502] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 06:13:11,824] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 06:31:34,675] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 06:31:35,035] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 06:47:43,915] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 06:47:44,180] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 07:09:27,065] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 07:09:27,441] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 07:22:20,143] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 07:22:20,325] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 07:33:10,169] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 07:33:10,475] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 07:47:54,457] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 07:47:54,764] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 08:12:11,582] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 08:12:11,905] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 08:29:00,400] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 08:29:00,742] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 08:38:37,564] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 08:38:37,843] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 08:47:58,022] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 08:47:58,386] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 09:11:29,607] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 09:11:29,913] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 09:26:24,852] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 09:26:25,155] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 09:35:38,712] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 09:35:38,895] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 09:48:04,239] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 09:48:04,539] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 10:09:46,383] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 10:09:46,693] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 10:29:35,025] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 10:29:35,317] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 10:38:51,568] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 10:38:51,914] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 10:47:42,862] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 10:47:43,070] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 11:09:28,096] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 11:09:28,517] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 11:21:25,081] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 11:21:25,309] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 11:33:30,165] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 11:33:30,465] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 11:47:40,714] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 11:47:40,903] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 12:16:48,423] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 12:16:48,777] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 12:49:34,482] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 12:49:34,688] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 13:11:45,631] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 13:11:45,929] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 13:26:39,360] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 13:26:39,712] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 13:34:07,696] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 13:34:08,069] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 13:47:53,950] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 13:47:54,256] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 14:09:15,621] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 14:09:15,841] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 14:26:48,269] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 14:26:48,638] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 14:35:36,821] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 14:35:37,165] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 14:47:40,915] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 14:47:41,294] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 15:10:11,858] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 15:10:12,143] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 15:25:44,794] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 15:25:45,166] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 15:33:09,369] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 15:33:09,577] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 15:47:50,717] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 15:47:51,035] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 16:12:09,960] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 16:12:10,302] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 16:30:59,060] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 16:30:59,280] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 16:47:55,108] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 16:47:55,322] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 17:08:25,969] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 17:08:26,334] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 17:25:11,350] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 17:25:11,674] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 17:33:04,690] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 17:33:04,868] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 17:47:33,686] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 17:47:34,020] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 18:12:45,049] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 18:12:45,358] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 18:30:35,326] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 18:30:35,494] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 18:47:39,102] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 18:47:39,270] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 19:07:56,389] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 19:07:56,654] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 19:20:40,428] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 19:20:40,642] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 19:33:05,701] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 19:33:06,061] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 19:47:46,830] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 19:47:47,153] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 20:09:49,949] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 20:09:50,247] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 20:24:06,194] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 20:24:06,498] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 20:34:09,857] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 20:34:10,131] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 20:47:29,183] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 20:47:29,381] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 21:08:53,212] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 21:08:53,543] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 21:22:14,040] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 21:22:14,347] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 21:33:44,909] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 21:33:45,131] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 21:47:51,382] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 21:47:51,656] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 22:09:40,567] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 22:09:40,789] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 22:28:15,069] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 22:28:15,381] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 22:37:07,329] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 22:37:07,604] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 22:47:56,974] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 22:47:57,300] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 23:10:24,862] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 23:10:25,201] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 23:24:22,236] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 23:24:22,568] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 23:33:11,630] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 23:33:12,052] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-04 23:47:48,318] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-04 23:47:48,626] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 00:41:26,899] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 00:41:27,124] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 01:44:54,424] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 01:44:54,731] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 02:21:46,204] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 02:21:46,416] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 02:43:26,142] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 02:43:26,417] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 02:50:35,909] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 02:50:36,182] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 03:12:53,622] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 03:12:53,946] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 03:30:00,823] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 03:30:01,002] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 03:39:06,235] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 03:39:06,549] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 03:47:34,900] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 03:47:35,246] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 04:13:59,239] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 04:13:59,502] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 04:31:39,868] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 04:31:40,148] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 04:47:49,101] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 04:47:49,378] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 05:09:21,406] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 05:09:21,752] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 05:27:12,380] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 05:27:12,659] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 05:35:58,713] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 05:35:59,022] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 05:47:40,472] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 05:47:40,796] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 06:13:09,265] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 06:13:09,464] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 06:32:05,911] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 06:32:06,187] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 06:47:41,832] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 06:47:42,122] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 07:09:03,960] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 07:09:04,243] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 07:22:47,135] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 07:22:47,433] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 07:33:44,812] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 07:33:45,120] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 07:47:54,199] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 07:47:54,375] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 08:12:02,497] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 08:12:02,811] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 08:28:30,585] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 08:28:30,889] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 08:38:08,382] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 08:38:08,586] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 08:47:40,886] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 08:47:41,298] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 09:12:09,795] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 09:12:10,110] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 09:28:22,894] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 09:28:23,326] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 09:36:50,854] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 09:36:51,024] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 09:47:49,313] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 09:47:49,495] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 10:09:56,095] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 10:09:56,408] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 10:29:23,772] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 10:29:24,082] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 10:38:25,921] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 10:38:26,220] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
[2024-06-05 10:47:32,633] 	INFO 	32 	id_data.py 	<module>() 	TC ID Data Started
[2024-06-05 10:47:32,959] 	ERROR 	104 	id_data.py 	<module>() 	Error occurred during TC ID Data process. Error: Dataframe is empty
Traceback (most recent call last):
  File "id_data.py", line 58, in <module>
    raise Exception('Dataframe is empty')
Exception: Dataframe is empty
